cli/container.go

```diff
@@ -232,6 +232,56 @@ func removeContainer(ID string) error {
 	return nil
 }
 
+// getContainerNetworks returns the networks a container is connected to
+func getContainerNetworks(ID string) (map[string]*network.EndpointSettings, error) {
+	ctx := context.Background()
+	docker, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	if err != nil {
+		return nil, err
+	}
+	options := types.ContainerListOptions{
```

Comment:
if the container ID is known, we can use it to filter for the container directly instead of later ranging over the full list of containers and comparing IDs, can't we?

Reply:
@iwilltry42 Yes, we could. Do you think it would be better than looking for labels? Because we must iterate with  `docker.ContainerList(ctx, ...)` anyway, right? I guess we could obtain 0 or 1 elements in this list, and that would be an improvement...

Reply:
Well, the ID of a container is unique, so if we know this, we don't need any further filtering and can even just use `docker.ContainerInspect` to get the details for exactly this container (or nothing if it doesn't exist)

Reply:
Gotcha. Fixing...